web/babel.config.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  "presets": [
-    '@babel/preset-env',
-    '@babel/preset-react',
-    '@babel/preset-typescript',
+  presets: [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-typescript"
   ],
-  "plugins": [
-    "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-proposal-optional-chaining"
-  ],
-}
\ No newline at end of file
+  plugins: [
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+    ["@babel/plugin-proposal-optional-chaining", { loose: true }]
```

Comment:
have to add `loose: true` or it errors out

Reply:
This seems fine if it passes the regression tests. We might split the babel config and use a different one for storybook later if we don't want to use loose mode.